test/env.sh

```diff
@@ -8,6 +8,8 @@
 : "${NUM_NODES:=2}"
 : "${COMMIT_HASH:=}"
 : "${ECR_REGISTRY:=}"
+: "${VPC_PUBLIC_SUBNET_STRING:=subnet-0921cea9bcffd7b77,subnet-0dbbc27abdf4a416e,subnet-0dec6ad34b32e791f}"
```

Comment:
Can these read from command line arguments instead of the hardcode?

Reply:
Thanks Sirish. Thats a really good point. I am removing the subnets and they can be set locally like ECR Registry.